sklearn/manifold/t_sne.py

```diff
@@ -142,10 +142,12 @@ def _kl_divergence(params, P, degrees_of_freedom, n_samples, n_components,
 
     # Q is a heavy-tailed distribution: Student's t-distribution
     n = pdist(X_embedded, "sqeuclidean")
-    n += 1.
```

Comment:
@AlexanderFabisch 
Regarding comparing the numerical and analytical gradient, -- I could implement this functionality into /sklearn/manifold/tests/test_t_sne perhaps? This is already included to check that it is correct for 1 degree of freedom! (which it passes of course)